packages/@aws-cdk/aws-iotevents/lib/expression.ts

```diff
@@ -65,11 +114,21 @@ class BinaryOperationExpression extends Expression {
     private readonly left: Expression,
     private readonly operator: string,
     private readonly right: Expression,
+    /**
+     * Indicates the priority of the operator.
+     * This is intended to be set according to MDN rules.
+     * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence#table
```

Comment:
I have adopted MDN as the operator priority rule. I think it can be used to whatever follow general rules and has concrete numbers.